modules/guides/pages/indexes.adoc

```diff
@@ -50,10 +50,4 @@ include::select-index.adoc[tags=intro]
 
 include::drop-index.adoc[tags=intro]
 
-* xref:drop-index.adoc[Dropping Indexes]
-
-== Index Advisor
-
-include::index-advisor.adoc[tags=intro]
-
-* xref:index-advisor.adoc[Index Advisor]
\ No newline at end of file
+* xref:drop-index.adoc[Dropping Indexes]
```

Comment:
As the Index Advisor is moving *out* of this section, isn't keeping the *link* here even more important now, as people might still be looking here for this content?